addons/sourcemod/scripting/superzombiefortress.sp

```diff
@@ -5615,6 +5597,69 @@ public MRESReturn Client_GetMaxHealth(int iClient, Handle hReturn)
 	return MRES_Ignored;
 }
 
+public MRESReturn Client_OnGiveNamedItem(int iClient, Handle hReturn, Handle hParams)
+{
+	static int iBlockedIndexes[] = {
+		57,  //Razorback
+		231, //Darwin's Danger Shield
+		642, //Cozy Camper
+		133, //Gunboats
+		444, //Mantreads
+		405, //Ali Baba's Wee Booties
+		608  //The Bootlegger
+	};
```

Comment:
Can `TF2Econ_GetItemSlot` be used to detect if weapon or cosmetic slot without needing this hardcode?

Reply:
`iBlockedIndexes` was not needed since we already block all secondary weapons